src/main/java/com/daml/quickstart/iou/ProjectionRunner.java

```diff
@@ -68,12 +71,13 @@ public static void main(String[] args) {
             ExecuteUpdate.create(
                     "insert into "
                         + projectionTable.getName()
-                        + "(contract_id, event_id, amount, currency) "
-                        + "values (?, ?, ?, ?)")
+                        + "(contract_id, event_id, amount, currency, json_data) "
+                        + "values (?, ?, ?, ?, ?::json)")
```

Comment:
java sql type doesn't have json so we have to do this cast on the db level

Reply:
I guess you could look into adding a `Bind.JsValue`

Reply:
We did this actually but the only solution possible will have that JsValue map to varchar/text in the sql types.

Reply:
That's fine, it just means that the user does not have to do 

```
                 .bind(5, jsonCodec.toJsValue(iou.data.toValue()).compactPrint(), Bind.String());
```

But instead can do

```
                 .bind(5, jsonCodec.toJsValue(iou.data.toValue()), Bind.JsValue());
```
Or maybe even we could do: (with 'implicit' json encoding)

```
                 .bind(5, iou.data, Bind.Value());
```
Which is very convenient for the user.

Reply:
So the naming might be a little confusion there, which can be improved, but that would be the idea.

Reply:
It's exactly the kind of thing `contramap` is good for.